src/utils/session-storage-utils.ts

```diff
@@ -1,8 +1,11 @@
 export const getFromSessionStorage = (key: string): string | null => {
-  const item = sessionStorage.getItem(key);
+  const item = localStorage.getItem(key);
```

Comment:
Please rename the function to reflect what it does (e.g. getFromLocalStorage) as the current name is misleading.